firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -132,7 +131,6 @@ class BrowserCoordinator: BaseCoordinator,
 
     func showHomepage() {
         let homepageController = self.homepageViewController ?? HomepageViewController(windowUUID: windowUUID)
-        homepageController.parentCoordinator = self
```

Comment:
no need for a coordinator for homepage since the navigation is done on browser view level and there are no navigation currently that is specific to homepage only.